src/movielens_dataset.jl

```diff
@@ -8,7 +8,7 @@
 # Both datasets have ratings in the range 1.0-5.0.
 
 function load_movielens_dataset(zipfile_name, archive_dir, split_ratio=0.10)
-    temp_dir = tempdir()
+    temp_dir = "." # tempdir()
```

Comment:
I'd like to keep tempdir() here. Can you revert this line or add some way of overriding it if you need a directory other than tempdir() (like passing a parameter temp_dir=nothing that is set to tempdir() if nothing is passed)?